README.md

```diff
@@ -596,6 +591,12 @@ More .NET Podcasts can be found on [The Sound of .NET](https://thesoundof.net/)
   * **Frequency** : Once every week
   * **Runtime**: 45 - 70 mins, regularly ~55 mins
 
+* [The freeCodeCamp Podcast](https://podcast.freecodecamp.org/) ([iTunes](https://itunes.apple.com/us/podcast/the-freecodecamp-podcast/id1313660749) | [Spotify](https://open.spotify.com/show/7CpuEnbCLIXwI6LEcbBOYP) | [Stitcher](https://www.stitcher.com/podcast/freecodecamp-podcast))
```

Comment:
It starts with `T` should probably move below

Reply:
The English language convention is to ignore the word "the" when alphabetically ordering things. People will be looking for this under the letter "f". Here's an example from Wikipedia: 
<img width="1172" alt="list_of_sovereign_states_-_wikipedia" src="https://user-images.githubusercontent.com/985197/52471566-2a07fc80-2bcc-11e9-8f7d-50e79b193fba.png">

Reply:
I think we might have to reorder the others, if this is the case I think

Reply:
OK - do you want me to go through and reorder them all in accordance with this convention?

Reply:
sure that should help. thanks